src/ProgressOnderwijsUtils/NullableAttributes.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.

```

Comment:
Needs comment explaining why this needs to be in our repo, ideally with link to docs.